handbook/company/development-groups.md

```diff
@@ -52,7 +52,7 @@ The goal of the customer experience (CX) group is to make users and customers ha
 | Engineering manager               | Sharon Katz
 | Quality assurance                 | Reed Haynes
 | Product manager                   | Zay Hanlon
-| Software engineers (developers)   | Jacob Shandling, Juan Fernandez*, Lucas Rodriguez, Rachel Perkins, Marcos Oviedo, Eric Shaw
+| Software engineers (developers)   | Jacob Shandling, Juan Fernandez*, Lucas Rodriguez, Rachel Perkins, Marcos Oviedo
```

Comment:
Eric is responsible for work on fleetdm.com for related product changes as part of CX, and also responsible for work on fleetdm.com that are marketing-driven rather than product-driven-- and/or are otherwise not part of a CX user story.

This way, anything that needs to span the website and the product goes through CX.  If it's purely a website change, then it goes to #g-website, since details about concurrent and other product changes don't matter.

Benefit of cross functional access > overhead of splitting time

Also allows Mike T to stay focused on website stuff, in terms of fresh wireframing and prioritization, and otherwise to be design escalation for marko/rachael/noah/mo/sales/marketing
<img width="904" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/73406a7d-e701-4daa-8fc7-5aa091437932">

Reply:
lmk if I'm missing anything or if you think we should clarify this further on the product groups page

Reply:
PS @eashaw == 50% CX, 50% website  (we should write that down, sorry I forgot to)